pkg/ebpf/tracer.go

```diff
@@ -111,9 +120,40 @@ func NewTracer(config *Config) (*Tracer, error) {
 		return nil, fmt.Errorf("could not start polling bpf events: %s", err)
 	}
 
+	go tr.expvarStats()
+
 	return tr, nil
 }
 
+func (t *Tracer) expvarStats() {
+	ticker := time.NewTicker(5 * time.Second)
+	// starts running the body immediately instead waiting for the first tick
+	for ; true; <-ticker.C {
+		stats, err := t.GetStats()
+		if err != nil {
+			continue
+		}
+
+		if states, ok := stats["state"]; ok {
+			if telemetry, ok := states.(map[string]interface{})["telemetry"]; ok {
```

Comment:
Do you really need the `states.(map[string]interface{})` cast?

Reply:
Yea, `state` is a nested map within a `map[string]interface{}`.